resources/qml/WelcomePages/CloudContent.qml

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2019 Ultimaker B.V.
+// Cura is released under the terms of the LGPLv3 or higher.
+
+import QtQuick 2.10
+import QtQuick.Controls 2.3
+
+import UM 1.3 as UM
+import Cura 1.1 as Cura
+
+
+//
+// This component contains the content for the "Ultimaker Cloud" page of the welcome on-boarding process.
+//
+Item
+{
+    UM.I18nCatalog { id: catalog; name: "cura" }
+
+    Label
+    {
+        id: titleLabel
+        anchors.top: parent.top
+        anchors.topMargin: 40
+        anchors.horizontalCenter: parent.horizontalCenter
+        horizontalAlignment: Text.AlignHCenter
+        text: catalog.i18nc("@label", "Ultimaker Cloud")
+        color: UM.Theme.getColor("primary_button")
+        font: UM.Theme.getFont("large_bold")
+        renderType: Text.NativeRendering
+    }
+
+    Column
+    {
+        anchors.top: titleLabel.bottom
+        anchors.topMargin: 80
+        anchors.horizontalCenter: parent.horizontalCenter
+
+        spacing: 60
+
+        Image
+        {
+            id: cloudImage
+            anchors.horizontalCenter: parent.horizontalCenter
+            source: UM.Theme.getImage("first_run_ultimaker_cloud")
+        }
+
+        Column
+        {
+            anchors.horizontalCenter: parent.horizontalCenter
+
+            spacing: 30
+
+            Label
+            {
+                id: highlightTextLabel
+                anchors.horizontalCenter: parent.horizontalCenter
+                horizontalAlignment: Text.AlignHCenter
+                text: catalog.i18nc("@text", "The next generation 3D printing workflow")
+                textFormat: Text.RichText
+                color: UM.Theme.getColor("text_light_blue")
```

Comment:
This theme entry makes no sense. Use primary colour?

Reply:
Done in 1e3945810